src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+       Machine [] mass = {new Bulldozer(), new Excavator(), new Truck()};
```

Comment:
do not use such naming, how to guess what it stores?
```suggestion
       Machine [] machines = {new Bulldozer(), new Excavator(), new Truck()};
```